packages/layout/src/types.ts

```diff
@@ -35,10 +27,28 @@ export interface BoxProps extends
   LayoutProps,
   PositionProps,
   SpaceProps,
-  TypographyProps {}
+  TypographyProps
+{
+  spaceX?: ResponsiveProp
+  sx?: ResponsiveProp
+  spaceY?: ResponsiveProp
+  sy?: ResponsiveProp
+
+  // This clashes with @types/react, which thinks it's a DOM attribute and
+  // wants it to be a string. This is a temporary workaround.
+  color?: string
+  textColor?: ResponsiveValue<Property.Color>
+}
```

Comment:
Check out [this article](https://stackoverflow.com/questions/41285211/overriding-interface-property-type-defined-in-typescript-d-ts-file) on how to override an extended interface by omitting a prop from the extended interface.